concourse/model/traits/release.py

```diff
@@ -152,6 +152,10 @@ class BuildstepFileAsset(Asset):
     `path` is interpreted relative to `step_output_dir`. Using Unix globbing, any matching files
     will be included (use `*` to include all, except files starting with period (`.`) character).
 
+    `prefix` will be removed from fnames in tar (similar to tar's -C option). Only effective in
```

Comment:
Wouldn't it be more convenient to allow overriding of `arcname`? 🤔

Reply:
then, we would have to provide a mapping for each single file. which - considering there are variations in filenames (e.g. in case of our package-build there is a version-number) - I think would be quite clunky / would mean one would have to use e.g. regex (sed-ish). imo, prefix is a lot easier and should be "good enough".

the arcnames can already be altered indirectly by simply renaming contained files 🤷🏻‍♂️